pkgs/tools/networking/openssh/default.nix

```diff
@@ -1,4 +1,5 @@
 { stdenv
+, pkgs
```

Comment:
I thought you should never put pkgs into inputs?

Reply:
this is rare, but correct. It is just like doing `buildPackages` explicitly, with the reason being to disable splicing.

Reply:
It'd be best to have an explanatory comment then.